pipeline/filters/aws-metadata.md

```diff
@@ -17,6 +17,9 @@ The plugin supports the following configuration parameters:
 | account\_id | The account ID for current EC2 instance. | false |
 | hostname | The hostname for current EC2 instance. | false |
 | vpc\_id | The VPC ID for current EC2 instance. | false |
+| tags_enabled | Specifies if should attach EC2 instance tags. EC2 instance must have the [instance-metadata-tags](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/ec2/modify-instance-metadata-options.html) option enabled (which is disabled by default). | false |
+| tags_include | Defines list of specific EC2 tag keys to inject into the logs. Tag keys must be separated by "," character. Tags which are not present in this list will be ignored. Example: "Name,tag1,tag2". | |
+| tags_exclude | Defines list of specific EC2 tag keys not to inject into the logs. Tag keys must be separated by "," character. If `tags_include` is not empty then `tags_exclude` has no effect. Example: "Name,tag1,tag2" | |
```

Comment:
I think the examples should be encased in code formatting, not quotes, since this might confuse the user into using quotes in their config file.

So: `Name,tag1,tag2`